InventoryCache/etc/di.xml

```diff
@@ -6,20 +6,20 @@
  */
 -->
 <config xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="urn:magento:framework:ObjectManager/etc/config.xsd">
```

Comment:
I cannot find changes from https://github.com/magento-commerce/inventory/pull/77/files#diff-e4815167c3a46544745397fd7e0d15fb3454b3ec023d4ca09cd0b00fc872422eR20
I cannot find all this changes in linked tickets

Reply:
Thank you, added